simplified-app-ekirjasto/src/main/java/fi/kansalliskirjasto/ekirjasto/EkirjastoDocumentStoreConfiguration.kt

```diff
@@ -18,7 +17,7 @@ class EkirjastoDocumentStoreConfiguration : DocumentConfigurationServiceType {
   override val feedback: DocumentConfiguration? =
     DocumentConfiguration(
       name = null,
-      remoteURI = URI.create("https://lib.e-kirjasto.fi/palaute/?lang=__LANGUAGE__&device_model=${Build.MANUFACTURER}%20${Build.MODEL}&software_version=${BuildConfig.VERSION_NAME}%20(${BuildConfig.VERSION_CODE})")
+      remoteURI = URI.create("https://lib.e-kirjasto.fi/palaute/?lang=__LANGUAGE__&device_model=${Build.MANUFACTURER}%20${Build.MODEL}&software_version=${BuildConfig.VERSION_NAME}%20(${BuildConfig.VERSION_CODE})".replace(" ", "%20"))
```

Comment:
Should probably use URL encoding (`URLEncoder.encode("string", "UTF-8")`) for all of the parameters, they could have other special characters, too.

And a space is kind of a bad separator for the device manufacturer and model, since the manufacturer's name can have a space as well (e.g. . For the version name and version code it's fine, but if the format is kept the same, the parenthesis should be URL encoded as well (`%28` and `%29`).

But instead of the current separators, it would be best to have each variable in it's own URL parameter (and to URL encode each of them), so `device_manufacturer=...&device_model=...&version_name=...&version_code=...`. Would that be a lot of work to change in the feedback form?

Reply:
Oh, and by the way, the Git commit hash would be really useful for developers. On Android, it should be available via `BuildConfig.SIMPLIFIED_GIT_COMMIT`, so you could add `commit=${BuildConfig.SIMPLIFIED_GIT_COMMIT}` (this is the only parameter that doesn't need URL encoding, since it's only hexadecimal characters).